frontend/server/src/Controllers/Course.php

```diff
@@ -2708,7 +2708,7 @@ public static function apiUpdate(\OmegaUp\Request $r): array {
             'finish_time',
             'school_id',
             'show_scoreboard' => ['transform' => function (string $value): int {
-                return $value == 'true' ? 1 : 0;
+                return boolval($value) ? 1 : 0;
```

Comment:
esto no debería ser

```suggestion
                return boolval($value);
```

así `show_scoreboard` se queda como booleano de verdad.

puedes también arreglar L2714 que esa en la misma situación?

Reply:
Listo, ya les agregué el `boolval` a las tres banderas que se mandan a la base de datos.